packages/dev/core/src/Loading/sceneLoader.ts

```diff
@@ -1453,6 +1539,7 @@ function importAnimationsSharedAsync(
 }
 
 /**
+ * @deprecated Please use the module level functions such as LoadAssetContainerAsync as they are more efficient for bundler tree shaking and allow plugin options to be passed through.

```

Comment:
Not sure it needs to be deprecated ?

Reply:
It would be nice to let folks know somehow that we don't intend to further invest in the class, and that if they want to benefit from improvements going forward they should switch to the module level functions. I thought a @deprecated was a good way to communicate that, but open to other ideas.

Reply:
Maybe we should make it clear it is not going away in the message ? I do not want them to be thinking they have to

Reply:
Updated, let me know if this one seems more clear!